GOVERNANCE.rst

```diff
@@ -0,0 +1,138 @@
+.. _governance:
+
+WarpX Governance
+================
+
+WarpX is led in an open governance model, described in this file.
+
+
+Steering Committee
+------------------
+
+Current Roster
+^^^^^^^^^^^^^^
+
+- Jean-Luc Vay (chair)
+- Remi Lehe
+- Axel Huebl
+
+See: `GitHub team <https://github.com/orgs/ECP-WarpX/teams/warpx-admins>`__
+
+Role
+^^^^
+
+The steering committee (SC) can change repo requirements, can do administrative operations such as rename/move/archive projects, etc.
```

Comment:
The term `repo requirements` may need to be clarified/defined.

Reply:
- repository settings
- branch protection rules